core/templates/dev/head/pages/exploration_player/continue_button_directive.html

```diff
@@ -0,0 +1,10 @@
+<md-button class="oppia-learner-confirm-button protractor-test-continue-to-next-card-button"
+           focus-on="<[::focusLabel]>"
+           ng-click="onClickContinueButton()"
+           style="margin-top: 12px;"
+           aria-label="Continue">
+  <span translate="I18N_PLAYER_CONTINUE_BUTTON"></span>
+  <i class="fa fa-arrow-right"
```

Comment:
Indent this and the next two lines by 2.

Reply:
Do you mean the "span" and "i" tags should be indented as per the attributes in "md-button"?

Reply:
Oh, sorry! Ignore this comment. I misread, the span tag is closed on the previous line.